spec/models/post_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'associations' do
```

Comment:
- [ ] You did a fantastic job testing the associations of the `Post` model. :+1:  I believe you have validations and custom methods inside the Post model which are not covered with the tests here. Kindly add tests for validations and custom methods in the Post model. You did a great job in the `user_spec`. It would be great if you do the same here. 🧐️